File: src/main/java/upbrella/be/user/dto/request/UpdateBankAccountRequest.java
package upbrella.be.user.dto.request;

import lombok.*;
import org.hibernate.validator.constraints.Range;

import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;

@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class UpdateBankAccountRequest {

    @Size(min = 1, max = 10)
    @NotBlank
    private String bank;
    @Size(min = 1, max = 45)
    @NotBlank
    private String accountNumber;
}

File: src/main/kotlin/upbrella/be/user/dto/request/UpdateBankAccountRequest.kt
package upbrella.be.user.dto.request

import upbrella.be.user.validation.OnlyNumbers
import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

data class UpdateBankAccountRequest(
    @field:Size(min = 1, max = 10)
    @field:NotBlank
    val bank: String,

    @field:Size(min = 1, max = 45)
    @field:NotBlank
    @field:OnlyNumbers
    val accountNumber: String
)